src/responsibleai/rai_analyse/gather_rai_insights.py

```diff
@@ -137,7 +138,7 @@ def main(args):
             args.dashboard, DashboardInfo.RAI_INSIGHTS_PARENT_FILENAME
         )
         with open(output_file, "w") as of:
-            json.dump(dashboard_info, of)
+            json.dump(dashboard_info, of, default=feature_metadata_to_dict)
```

Comment:
similar, can we have some handler that is named more generally?

Reply:
yes! updating!